### DISCUSSION TITLE
SCSI Activity LED?

### DISCUSSION THREAD

**Discussion by mg-man** [erichelgeson/BlueSCSI-v1#45] [Q&A] [answered]
Apologies if I'm missing the obvious... but is there a way to connect an external LED to show "HD" activity?  I believe the Blue Pill onboard "user led" is being driven this way?

**Comment by erichelgeson** [comment, marked as answer]
Hi @mg-man ! yes there is - take a look at the instructions in the wiki and the next version will use a dedicated pin. 

https://github.com/erichelgeson/BlueSCSI/wiki/External-LED

**Reply by mg-man** [reply]
HI Eric, I had been considering 'replacing' the on-board LED on the BluePill...  I do think it'd be a good idea to add this to a future version since removing the existing LED is pretty delicate work.  Cheers, Lee

**Reply by mg-man** [reply]
> the next version will use a dedicated pin

I see the a 1.1-a version in the works... any idea on when it'll be available?  I'm considering picking up a couple more...

**Reply by erichelgeson** [reply]
Well probably right after you order some boards haha! Honestly delayed due to family and summer so I expect fall. Socket your bluepills and you can always upgrade if you are really are hesitant. Pcbs are one of the cheaper parts.

**Reply by nglayton** [reply]
1.1-a version is out.......Do we have pins for the external LED? 
Looks a lot like v1.0 except the SDcard slot is turned 90 degrees.

**Reply by erichelgeson** [reply]
> 1.1-a version is out.......Do we have pins for the external LED? Looks a lot like v1.0 except the SDcard slot is turned 90 degrees.

You can find the changes/additions in the repo's readme https://github.com/erichelgeson/bluescsi#1-1-a-latest or open the files up in kicad. To your question, yes.
